react/ui-components/src/hoc/UploadAndDownloadDocumentHandler.js

```
@@ -0,0 +1,316 @@
+        tenantId: tenantId,
+        schemaCode: "Studio.ServiceConfiguration",
+            filters:{
+              module:module
+            }
+          },
+        },
+      };
+ const { isLoading, data:serviceconfig } = Digit.Hooks.useCustomAPIHook(serviceconfigrequestCriteria);
+
+  const downloadPdf = (blob, fileName) => {
+    if (window.mSewaApp && window.mSewaApp.isMsewaApp() && window.mSewaApp.downloadBase64File) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)